rust/gel-http/src/cache.rs

```diff
@@ -65,22 +94,30 @@ impl Cache {
             },
             cache: Arc::new(Mutex::new(CacheItems::new(
                 NonZero::new(100).unwrap(),
-                1024 * 1024,
+                10 * 1024 * 1024,
```

Comment:
Just checking: this increases the cache max size to 10MB right?

Reply:
Correct. TBH I don't understand _why_ that cache request was falling out of the LRU (I wonder if there was a second race?), but I think it's reasonable to hold up to 10MB of cached JWK keys _just in case_ that was somehow related.